snippets/ansible/login.yml

```diff
@@ -0,0 +1,22 @@
+---
+  - name: login
+    uri:
+      url: "{{baseurl}}login"
+      method: POST
+      body_format: json
+      status_code: 201
+      headers:
+        content-type: application/vnd.netbackup+json;version=3.0
+      body:
+        userName: "{{username}}"
+        password: "{{password}}"
+      validate_certs: no
+      return_content: yes
+    register: login
+
+  - name: set facts
+    set_fact:
+      login_token: "{{login.json.token}}"
```

Comment:
Some plays use `login_token`, and others use `login.json.token` directly. What's the difference?